packages/charts/src/chart_types/goal_chart/specs/index.ts

```diff
@@ -54,11 +54,12 @@ export interface GoalSpec extends Spec {
   tickValueFormatter: GoalLabelAccessor;
   labelMajor: string | GoalLabelAccessor;
   labelMinor: string | GoalLabelAccessor;
-  centralMajor: string | GoalLabelAccessor;
+  centralMajor: string | GoalLabelAccessor | (() => string);
```

Comment:
Same here but also this would be a duplicated union of `(() => string) | (() => string)`.

```suggestion
  centralMajor: string | GoalLabelAccessor;
```

Reply:
9e61b47 for changes thanks!